templates/etc/systemd/system/cerebro.service.erb

```diff
@@ -1,3 +1,8 @@
+<% if !@address.nil? -%>
+  <%- bind_address = "-Dhttp.address=#{@address}" -%>
+<% else -%>
+  <%- bind_address = undef -%>
```

Comment:
This line causes a syntax error. Is this not `undef` but `nil`?

Reply:
@yano3 You're right... Fixed..